react/training/src/app/screens/Login/index.js

```diff
@@ -0,0 +1,15 @@
+import React, { Component } from 'react';
+
+import LoginForm from './components/layout';
+
+class Login extends Component {
+  submit = values => {
+    window.alert(JSON.stringify(values, null, 4));
+  };
+
+  render() {
+    return <LoginForm onSubmit={this.submit} />;
```

Comment:
`Login/index.js` should call `Login/layout.js`. `layout.js is de dumb part of the component with all html and styles.`
Then in your `layout.js` you have to call to `LoginForm/Index.js`. `LoginForm` is another component. Should be inside in `Login/Components`